src/jit/target.h

```diff
@@ -1440,7 +1440,7 @@ typedef unsigned short regPairNoSmall; // arm: need 12 bits
 
   // The registers trashed by the CORINFO_HELP_STOP_FOR_GC helper
   // See vm\arm\amshelpers.asm for more details.
-  #define RBM_STOP_FOR_GC_TRASH     (RBM_CALLEE_TRASH & ~(RBM_FLOATRET | RBM_INTRET))
+  #define RBM_STOP_FOR_GC_TRASH     (RBM_CALLEE_TRASH & ~(RBM_DOUBLERET | RBM_INTRET))
```

Comment:
> Looking at vm\arm\amshelpers.asm, it appears that JIT_RareDisableHelper preserves d0-d3. I'm not sure whether that translates to our f0-f3, or whether that is actually f0-f7, but in any case this doesn't seem consistent with that.

Yes, and `JIT_RareDisableHelper` defined in asmhelpers.S also preserves d0-d3 as below.
https://github.com/dotnet/coreclr/blob/master/src/vm/arm/asmhelpers.S#L1013-L1031

> I'm not sure whether that translates to our f0-f3, or whether that is actually f0-f7, but in any case this doesn't seem consistent with that.

Are you suggesting that we had better exclude all d0-d3 from `RBM_STOP_FOR_GC_TRASH` ?

> Also, even though the above comment is the same as for the other architectures, it would have eased my search if it said:

Sure, I will update the comment :)
Frankly speaking, I also had searched for the function.